src/parser/list_paragraph_parser.py

```diff
@@ -6,10 +6,13 @@ def parse_ordered_list(content) -> (int, int, 'OrderedList'):
     """Parse a list paragraph, and return (begin, end, OrderedList) if parseable,
     or (-1, -1, None) that no such a element
     """
-    pattern = r'^[ ]*\d\.[ ]+(.*)$\n*'
+    pattern = r'^([ \t]*)\d\.[ ]+(.*)$\n*'
```

Comment:
Thought we agreed on replace tab with 4 spaces before matching?

Reply:
Addressed

Reply:
Then we probably don't need to match `\t` anymore in this regex

Reply:
lol you’re right